index.html

```diff
@@ -269,117 +269,101 @@ <h2 class="font-work-sans font-semibold pl-8 text-2xl text-center">
                         class="w-32 flex flex-col justify-center items-center rounded-xl font-work-sans p-2 cursor-pointer"
                     >
                         <img
-                            src="./resources/assets/jsloversLogo.png"
-                            alt="jslovers homepage"
-                            width="100"
+                            src="./resources/assets/microsoft.png"
+                            class="company_logo"
```

Comment:
We don't need to add custom CSS. We are using tailwind. We can add tailwind classes here